### PR TITLE
Specify signing identity in json rpc signAndPost call

### LINKED ISSUE #1315: Add signing identity to signAndPost RPC call
Follow-up to #1308

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #1315

### CODE REVIEW

File: packages/iov-multichain/src/jsonrpcsigningserver.spec.ts [hunk 1]
```diff
@@ -247,6 +247,7 @@ describe("JsonRpcSigningServer", () => {
       method: "signAndPost",
       params: {
         reason: "string:Please sign",
+        identity: TransactionEncoder.toJson(signer),
```
Comment (webmaster128) [inline packages/iov-multichain/src/jsonrpcsigningserver.spec.ts]: Can we call this `signer` and update https://github.com/iov-one/iov-core/blob/master/docs/out-of-process-rpc.md?

Comment (willclarktech) [reply packages/iov-multichain/src/jsonrpcsigningserver.spec.ts]: Updated. It's a little confusing that we also use `signer` in lots of examples to refer to an instance of `MultiChainSigner` though.

Comment (webmaster128) [reply packages/iov-multichain/src/jsonrpcsigningserver.spec.ts]: Hmm, yes, right, good point. The tool and the tool's user have the same name. I don't see an easy way to resolve this right now.

Comment (willclarktech) [reply packages/iov-multichain/src/jsonrpcsigningserver.spec.ts]: Red signer and blue signer?

Comment (webmaster128) [reply packages/iov-multichain/src/jsonrpcsigningserver.spec.ts]: "Open signer" would be good for a tool that is open to be used by anybody, such as closed signers.

**Pull request merged**